selenium_tests/lti_emailer/test_setup.py

```diff
@@ -0,0 +1,41 @@
+__author__ = 'hic048'
+import unittest
+from ddt import ddt, data, unpack
+
+from selenium_tests.lti_emailer.emailer_base_test_case import EmailerBaseTestCase
+from selenium.webdriver.support.ui import Select
+from selenium_tests.lti_emailer.page_objects.add_user_po import UserAdd, Locator3
+from selenium_tests.lti_emailer.emailer_base_test_case import CANVAS_ROLES
+from selenium_tests.base_test_case import get_xl_data
+
+# BASE_URL = /courses/6389/external_tools/1759
+#URL = "http://canvas.icommons.harvard.edu/courses/6389/users"
+
+@ddt
+class Addingusers(EmailerBaseTestCase):
+    # This adds a list of predefined users to specific roles according to /test data/roles.xlsx.
+    # If this list of users need to be populated on another course, locator "_add_people_url" needs
+    # to be updated. For testing with large data add, please refer to team wiki.
+    @data(*get_xl_data(CANVAS_ROLES))
+    @unpack
+    def test_user_setup(self, user_id, roles):
+        for n in self, user_id, roles:
+            driver = self.driver
+            print "adding " + user_id + "to " + Locator3._add_people_url
+            adduser = UserAdd(driver) #instantiate
```

Comment:
As mentioned in a comment on add_user_po.py, I think we could be encapsulating things a bit better -- a PageObject is meant to expose only page actions, so we would load the `UserAdd` (probably want to call it something like `CanvasCourseUsersPage` instead) and then do something like `userPage.add_user(user_id)` -- all the `send_keys`, `find_elements`, etc should be encapsulated in the PageObject. It makes this much smaller, and makes changes to the  `CanvasCourseUsersPage` only require updating a single location.